src/main/java/seedu/address/model/person/Major.java

```diff
@@ -10,48 +10,75 @@
 public class Major {
 
     public static final String MESSAGE_CONSTRAINTS = "Majors should not be blank and must be a valid"
-            + " major offered at NUS.";
+            + " major offered at NUS. Valid major list can be found in the user guide.";
```

Comment:
add smth like "u can find the user guide by using the `help` command

Reply:
ok, added. thanks!